externs/shaka/player.js

```diff
@@ -986,6 +1009,10 @@ shaka.extern.ManifestConfiguration;
  *   The maximum time a variant can be disabled when NETWORK HTTP_ERROR
  *   is reached, in seconds.
  *   If all variants are disabled this way, NETWORK HTTP_ERROR will be thrown.
+ * @property {boolean} parsePrftBox
+ *   If <code>true</code>, will raise a shaka.extern.ProducerReferenceTime
```

Comment:
This doesn't mention the event name that apps would need to listen for.

Reply:
Good Spot!